src/gui/downloadfromurldlg.h

```diff
@@ -31,23 +31,31 @@
 #ifndef DOWNLOADFROMURL_H
 #define DOWNLOADFROMURL_H
 
+#include <QClipboard>
 #include <QDialog>
 #include <QMessageBox>
-#include <QString>
+#include <QPushButton>
 #include <QRegExp>
+#include <QString>
 #include <QStringList>
-#include <QClipboard>
+
 #include "ui_downloadfromurldlg.h"
 
-class downloadFromURL : public QDialog, private Ui::downloadFromURL{
+class downloadFromURL : public QDialog, private Ui::downloadFromURL
+{
   Q_OBJECT
 
   public:
-    downloadFromURL(QWidget *parent): QDialog(parent) {
+    downloadFromURL(QWidget *parent): QDialog(parent)
+    {
       setupUi(this);
       setAttribute(Qt::WA_DeleteOnClose);
       setModal(true);
 
+      buttonBox->button(QDialogButtonBox::Ok)->setText(tr("Download"));
+      connect(buttonBox, &QDialogButtonBox::accepted, this, &downloadFromURL::downloadButtonClicked);
+      connect(buttonBox, &QDialogButtonBox::rejected, this, &QDialog::rejected);
```

Comment:
I suggested to connect it to QDialog::reject slot, not to QDialog::rejected signal.

Reply:
typo, fixed.